src/pages/UserProfile/SelfProfile/SelfProfilePage.tsx

```diff
@@ -40,25 +30,25 @@ export default function SelfProfilePage(): JSX.Element {
       <div>
         <img
           className="h-32 w-full mx-auto object-cover lg:h-48"
-          src={userCoverImage}
+          src={userInfo?.coverImage}
```

Comment:
Should we use a default cover image?

Reply:
this is an onlyauth route, if it is undefined you cannot access, undefined image is a transparent image, but added